### PR TITLE
[ENHANCEMENT] Ping discord user instead of discord id

### PR DESCRIPTION
**Pull request by Bama94**
This changes the log of players joining. It's gives the `@playersdiscordname` instead of showing `discord:astringofnumbers`

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes
- Does your code fit the style guidelines? Yes
- Does your PR fit the contribution guidelines? Yes

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -88,6 +119,7 @@ end)
 
 RegisterNetEvent('qb-multicharacter:server:loadUserData', function(cData)
     local src = source
+    local identifierlist = ExtractIdentifiers(src)
```
Comment (Z3rio) [inline server/main.lua:122]:
```suggestion
```
Not needed

File: server/main.lua [hunk 2]
```diff
@@ -96,7 +128,7 @@ RegisterNetEvent('qb-multicharacter:server:loadUserData', function(cData)
         QBCore.Commands.Refresh(src)
         loadHouseData(src)
         TriggerClientEvent('apartments:client:setupSpawnUI', src, cData)
-        TriggerEvent("qb-log:server:CreateLog", "joinleave", "Loaded", "green", "**".. GetPlayerName(src) .. "** ("..(QBCore.Functions.GetIdentifier(src, 'discord') or 'undefined') .." |  ||"  ..(QBCore.Functions.GetIdentifier(src, 'ip') or 'undefined') ..  "|| | " ..(QBCore.Functions.GetIdentifier(src, 'license') or 'undefined') .." | " ..cData.citizenid.." | "..src..") loaded..")
+        TriggerEvent("qb-log:server:CreateLog", "joinleave", "Loaded", "green", "**".. GetPlayerName(src) .. "** ("..("<@"..identifierlist.discord:gsub("discord:", "")..">" or 'undefined') .." |  ||"  ..(QBCore.Functions.GetIdentifier(src, 'ip') or 'undefined') ..  "|| | " ..(QBCore.Functions.GetIdentifier(src, 'license') or 'undefined') .." | " ..cData.citizenid.." | "..src..") loaded..")
```
Comment (Z3rio) [inline server/main.lua:131]:
```suggestion
        TriggerEvent("qb-log:server:CreateLog", "joinleave", "Loaded", "green", "**".. GetPlayerName(src) .. "** ("..("<@"..QBCore.Functions.GetIdentifier(src, 'discord'):gsub("discord:", "")..">" or 'undefined') .." |  ||"  ..(QBCore.Functions.GetIdentifier(src, 'ip') or 'undefined') ..  "|| | " ..(QBCore.Functions.GetIdentifier(src, 'license') or 'undefined') .." | " ..cData.citizenid.." | "..src..") loaded..")
```
Revert to old / default GetIdentifier function

**Review by Z3rio — CHANGES_REQUESTED** [review]
This is a great idea, although you have added alot of stuff that isnt needed, please dont add functions like "ExtractIdentifiers" when there already is a function in qb-core to do such a thing.

File: server/main.lua (hunk 2 above)
Comment (Bama94) [reply server/main.lua:131]: I did not think about that! thank you

**Review by Z3rio — APPROVED** [review]
LGTM

**Pull request merged**